uploader.json

```diff
@@ -182,6 +182,11 @@
             "pattern": [],
             "symbolic_name": "facter"
         },
+	{
+	    "command": "/usr/bin/qpid-stat -g --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b amqps://localhost:5671",
```

Comment:
@TripleJQK your indention is off here, if wouldn't mind to fix it here and in the v2 json as well, this will be ready to go.